### ISSUE TITLE
Android App - Text Element - Font family dropdown selected state is wrong color

### ISSUE THREAD

**Issue by flukeout** [mozilla/webmaker-core#430]
The selected option in the font family dropdown has a white indicator instead of black. See below. It looks weird...

![image](https://cloud.githubusercontent.com/assets/25212/8946053/fff4df00-3541-11e5-8b81-04b4e0679421.png)

**Comment by gvn** [comment]
Note: this is part of Android's UI / shadow DOM(?).

**Comment by gvn** [comment]
@thisandagain any thoughts on fixing this?

**Comment by thisandagain** [comment]
Ooof. That's going to be a tough one as it's a part of the underlying Chromium view. Most likely easiest to fix by writing / integrating a drop-down component in `core`.

**Comment by gvn** [comment]
Yeah, it's possible our styles are affecting it, but hard to say. I wasn't able to inspect it in dev tools.

Recommend pushing this to next sprint... cc: @xmatthewx

**Comment by xmatthewx** [comment]
Yes. We can assess later.
